### PR TITLE
Implement profile editing

### PR DESCRIPTION
**Pull request by stevenpi**
Screenshot of editing page:

profile-edit

### CODE REVIEW

**Comment by mbaechtold** [comment]
I'll have a look at this when #29 has been merged.

File: Blog/templates/Blog/user_detail.html [hunk 1]
```diff
@@ -6,19 +6,22 @@
...
-                    <img src="{{ user.profile.image.url }}" style="border-radius: 50%"/>
+                {% if user_instance.profile.image %}
+                    <img src="{{ user_instance.profile.image.url }}" style="border-radius: 50%"/>
                 {% else %}
                     <img src="{% static 'images/default_user.png' %}"/>
                 {% endif %}
             </div>
-            <h1>{% trans 'Username' %}: {{ user }}</h1>
+            <h1>{% trans 'Username' %}: {{ user_instance }}</h1>
+        {% if user.is_authenticated %}
```
Comment (mbaechtold) [inline Blog/templates/Blog/user_detail.html]: The currently logged in user is available on the request: `request.user`. If you use this, you can revert the renaming of the context variable `user` to  `user_instance`.

**Comment by stevenpi** [comment]
Adapted the code. Also fixed another bug, which let the "Edit Profile" button show everytime a user was logged in

File: Blog/templates/Blog/user_detail.html [hunk 2]
```diff
@@ -6,19 +6,24 @@
     <div class="row">
         <div class="cell-md-4 cell-sd-full">
             <div class="img-container profile-picture">
-                {% if user.profile.image %}
-                    <img src="{{ user.profile.image.url }}" style="border-radius: 50%"/>
+                {% if user_instance.profile.image %}
```
Comment (mbaechtold) [inline Blog/templates/Blog/user_detail.html]: If I'm not mistaken, `user_instance` is not available in the context.

**Comment by stevenpi** [comment]
Adapted code as requested

**Pull request merged**